openembedded-core-mbl/meta/classes/mbl-logrotate-config.bbclass

```diff
@@ -169,7 +180,11 @@ def mbl_logrotate_config_write(d, log_name):
         file.write(" {\n")
 
         for key, value in normal_fields.items():
-            file.write("{}{} {}\n".format(indent, key, value))
+            if key is 'noarg_opt':
```

Comment:
Do you really mean `is` rather than `==` here? IIUC, `is` checks for object identity rather than equality.

Reply:
Addressed in bf33d4f